### DISCUSSION TITLE
GML support

### DISCUSSION THREAD

**Discussion by charlieduzstuf** [fgsoftware1/fgengine#9] [Ideas]
https://github.com/maiple/opengml
https://github.com/topics/gml
http://github.com/zbanack/c-to-gml
http://github.com/davidyu/gml-cpp
https://www.youtube.com/watch?v=P08i1kCmu3Y
Like Lua and is in a very good but hard engine

**Comment by fgsoftware1** [comment]
> https://github.com/maiple/opengml https://github.com/topics/gml Like Lua and is in a very good but hard engine

interpreted lang
